homeassistant/components/file/config_flow.py

```diff
@@ -99,8 +98,7 @@ async def async_step_sensor(
             if not await self.validate_file_path(user_input[CONF_FILE_PATH]):
                 errors[CONF_FILE_PATH] = "not_allowed"
             else:
-                name: str = user_input.get(CONF_NAME, DEFAULT_NAME)
-                title = f"{name} [{user_input[CONF_FILE_PATH]}]"
+                title = f"{DEFAULT_NAME} [{user_input[CONF_FILE_PATH]}]"
```

Comment:
I think we can use the name in the imported config but not ask for name in the flow.

Reply:
This hasn't been addressed.

Reply:
It is, address as this is only for new entries, and we do not ask in the flow.

Reply:
Ok, thanks.